static/src/javascripts/projects/common/modules/onward/more-tags.js

```diff
@@ -9,15 +9,16 @@ define([
     $,
     bean
 ) {
+    var ACTIVE_STATE = 'is-available';
 
     function MoreTags() {
         this.init = function () {
             var $more = $('.js-more-tags');
             if ($more.length !== 0) {
-                $more.removeClass('modern-hidden');
+                $more.addClass(ACTIVE_STATE);
```

Comment:
This flips the responsibility for displaying the more tag onto the component itself rather than implying on the implicit behaviour of `modern-hidden`.